### PR TITLE
[PTQ][OV] Remove IS calculation from `calibrate.py`, move it to separate tool

### PR DESCRIPTION
**Pull request by daniil-lyakhov**
### Changes

[PTQ][OV] Remove IS calculation from `calibrate.py`, move it to separate tool

### Reason for changes

To make calibration clearer

### CODE REVIEW

**Review by alexsu52 — APPROVED** [review]
LGTM

File: tools/get_ir_is_by_model_id.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
+"""
```
Comment (andrey-churkin) [inline tools/get_ir_is_by_model_id.py:1]: Could we give a more readable name for this tool than `get_ir_is_by_model_id.py?`

File: tools/get_ir_is_by_model_id.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+                continue
+            next_nodes = get_next_nodes(node, 0)
+            assert len(next_nodes) == 1
+            next_node = next_nodes[0]
+            assert next_node.type_info.name == 'Convolution'
+            return [next_node.get_friendly_name()]
+    return None
+
+
+IS_TO_BUILDER_MAP = {
```
Comment (andrey-churkin) [inline tools/get_ir_is_by_model_id.py:91]: Could you please use a full name i.e. ignored scope instead of `is`?

Comment (daniil-lyakhov) [reply tools/get_ir_is_by_model_id.py]: Done, please check

**Review by andrey-churkin — APPROVED** [review]

**Pull request merged**